# The tasks to be added to the executive system
Task[] tasks
---
uint64[] task_ids